src/main/cpp/include/utils/alleles_combiner.h

```diff
@@ -219,7 +327,7 @@ class AllelesCombiner {
     //Also, we only store the samples which actually have a deletion/MNV - not all samples.
     //Contains row indexes of samples with new data at current iteration and containing deletions/MNVs. By tracking
     //these indexes, we can determine which rows must insert spanning deletions in the subsequent locations
-    std::vector<int64_t> m_row_query_idx_with_deletion_MNV_at_current_location_vec;
+    std::vector<size_t> m_row_query_idx_with_deletion_MNV_at_current_location_vec;
```

Comment:
Are we standardizing on `size_t` instead of `int64_t`?

Reply:
For row values - int64_t
For row_query_idx - size_t